ts/api/pagopa.ts

```diff
@@ -134,12 +134,80 @@ type GetWalletsUsingGETExtraT = MapResponseType<
   WalletListResponse
 >;
 
+/**
+ *
+ * sanitaze psp tags if it is an array with values (avoid no string value and duplicates)
+ * otheriwise set tags as an empty array
+ * @param w the wallet object
+ */
+const fixWalletPspTagsValues = (w: any) => {
+  if (Array.isArray(w.psp.tags)) {
```

Comment:
`w.psp` may be `null` or `undefined`, thus `w.psp.tags` will throw an exception - you should use an `io-ts` decoder here